backend/src/modules/api/api.controller.ts

```diff
@@ -30,6 +31,11 @@ export class ApiController {
         return this.programService.getByGuid(params.guid);
     }
 
+    @Get('document/:guid')
+    getDocumentByGuid(@Param() params): Promise<string>{
+        return this.documentService.getByTitle(params.guid);
```

Comment:
Hi @TheBroMoe, it looks like you forgot to change params.guid to params.title and getDocumentByGuid to getDocumentByTitle.

Reply:
Hi @CodyGramlich, Thanks for letting me know, I'll change it right away!